polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -44,64 +44,58 @@ use sp_version::RuntimeVersion;
 use frame_support::weights::ConstantMultiplier;
 pub use frame_support::{
 	construct_runtime,
+	dispatch::DispatchClass,
 	pallet_prelude::Get,
 	parameter_types,
-	traits::{EitherOfDiverse, Everything, IsInVec, Nothing, PalletInfoAccess, Randomness},
+	traits::{
+		EitherOfDiverse, Everything, IsInVec, Nothing, PalletInfoAccess, Randomness,
+		WithdrawReasons,
+	},
 	weights::{
-		constants::{BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight, WEIGHT_PER_SECOND},
-		DispatchClass, IdentityFee, Weight,
+		constants::{BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight},
+		IdentityFee, Weight,
 	},
 	PalletId, RuntimeDebug, StorageValue,
 };
 use frame_system::{
 	limits::{BlockLength, BlockWeights},
 	EnsureRoot,
 };
+
+pub use parachains_common as common;
+use parachains_common::{
+	opaque, AccountId, AuraId, Balance, BlockNumber, Hash, Header, Index, Signature,
+	AVERAGE_ON_INITIALIZE_RATIO, DAYS, HOURS, MAXIMUM_BLOCK_WEIGHT, MINUTES, NORMAL_DISPATCH_RATIO,
+	SLOT_DURATION,
+};
```

Comment:
Before, we redefined the types redundantly in both runtimes. This is also how they do it upstream now.